src/invoke.rs

```diff
@@ -275,29 +272,19 @@ impl Cmd {
         let fee: u32 = 100;
         let sequence = account_details.sequence.parse::<i64>()?;
 
-        // Get the contract
-        let wasm = if let Some(f) = &self.wasm {
-            // Get the contract from a file
-            // TODO: as a workaround (RPC server doesn't yet implement getContractData)
-            //       we allow supplying the contract in the commandline
-            //       we should consider removing this later on
-            fs::read(f).map_err(|e| Error::CannotReadContractFile {
-                filepath: f.clone(),
-                error: e,
-            })?
-        } else {
-            // Get the contract from the network
-            let contract_data = client
-                .get_contract_data(
-                    &hex::encode(contract_id),
-                    ScVal::Static(ScStatic::LedgerKeyContractCode),
-                )
-                .await?;
-
-            match ScVal::from_xdr_base64(contract_data.xdr)? {
-                ScVal::Object(Some(ScObject::Bytes(bytes))) => bytes.to_vec(),
-                scval => return Err(Error::UnexpectedContractCodeDataType(scval)),
```

Comment:
Can you make --wasm optional rather than a conflict to provide? There are edge cases where we might actually want to provide the wasm on the command line rather than rely on it being downloaded. For e.g. the token contract. It will also mean this change won't be a breaking change in the final hour.

Reply:
Can you elaborate on the edge case and why downloading the contract is not an option?

Reply:
Regarding breaking changes, we see not at a point of considering backwards compatibility when invocation support was added 3 days ago.

Reply:
Almost every PR has been a breaking change in the past few days (starting with multiple unconsensuated flag renamings)

Reply:
In the last day we started trying to avoid breaking changes that would affect the crowdfund example dapp's scripts and the e2e scripts. That's what I was referring to.